index.html

```diff
@@ -14,31 +16,33 @@
         <canvas class="game_menu" width="780" height="450"></canvas>
         <canvas class="field" width="410" height="450"></canvas>
         <form>
-            <input type="button" value="Run">
             <textarea cols="30" rows="20"></textarea>
+            <input type="button" value="Run">
         </form>
     </div>
     <div class="instructions">
-        <h2>Instructions</h2>
+        <h1>Instructions</h1>
         <p>
             To control the robot you have to give it programming instructions.
             The available instructions are:
         </p>
-        <code>
-            robot.moveRight(int steps);<br>
-            robot.moveLeft(int steps);<br>
-            robot.moveUp(int steps);<br>
-            robot.moveDown(int steps);<br>
-            robot.pushButton();<br>
-            robot.openChest();<br>
-            robot.hasRedKey();<br>
-            robot.hasGreenKey();<br>
-            robot.hasBlueKey();<br>
-            if (robot.has&lt;Color&gt;Key()) {<br>
-            &nbsp;&nbsp;&nbsp;&nbsp;robot.&lt;SomeMethod&gt;();<br>
-            }
-            robot.openDoor();
-        </code>
+        <div class="info-area">
```

Comment:
This `<div>` is not needed. Just put the class directly on the `<code>` element instead